packages/share/android/src/main/java/io/flutter/plugins/share/SharePlugin.java

```diff
@@ -58,4 +67,26 @@ private void share(String text) {
       mRegistrar.context().startActivity(chooserIntent);
     }
   }
+
+  private void shareFile(String uri, String mimeType) {
```

Comment:
This code appears to work no matter the scheme of the uri (ie. it doesn't have to be `file://`). Is that correct? If so, can we generalize the API also on the Dart side to `shareUri`?

Reply:
Quick test on my end, it doesn't work. Sharing a `file://` image to Signal works, but sharing a `https://` image doesn't.

Which one meshes better with iOS sharing?